packages/astro/src/default/utils/url.ts

```diff
@@ -0,0 +1,18 @@
+export function joinPaths(baseURL: string, ...paths: string[]): string {
```

Comment:
Can't we use something like `URL()` to join the paths?

If not, I'd like to see some tests for this method 🙈 .

Reply:
I thought about that at first but wasn't sure it would work. Totally agree on adding tests. Will do that now :+1:

Reply:
Oh nice after adding tests I found a bug (not a current bug in our code but one that could eventually happen in the future). It's when all the arguments are empty strings.

Reply:
In what scenario's wouldn't something like `new URL()` work? I'd rather use something like that then a custom implementation.

Reply:
No it doesn't because baseURL is not a baseURL. Oh gosh I just realised that the first argument is named incorrectly. I'll fix that :+1: